app/test_main.py

```diff
@@ -1,3 +1,45 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_if_years_are_integer() -> None:
+    cat_years = 26
+    dog_years = 25
+    assert isinstance(cat_years, int) and isinstance(dog_years, int)
+
+
+def test_if_years_are_positive() -> None:
+    cat_years = 26
+    dog_years = 25
+    assert cat_years > 0 and dog_years > 0
```

Comment:
This is always true) You must add tests when you call function get_human_age() with args <= 0 also if you pass another type than int